src/clockgen/clockgen.c

```diff
@@ -242,7 +242,7 @@ s32 sceClockgenSetSpectrumSpreading(s32 mode)
 }
 
 //0x000002B4
-s32 _sceClockgenModuleStart(SceSize args __attribute__((unused)), void *argp __attribute__((unused)))
```

Comment:
Looking at the SDK, the prototype for module_start appears to be this:
```c++
int module_start( SceSize argSize, const void *argBlock );
```

So perhaps we should rather change that definition than all these instances of `module_start`:
https://github.com/uofw/uofw/blob/f5419041056051febd3f56596f802119a942d83e/include/common/module.h#L141

Reply:
Fixed